src/client/inputhandler.h

```diff
@@ -456,10 +384,8 @@ class RandomInputHandler : public InputHandler
 		mousepos += mousespeed;
 	}
 
-	s32 Rand(s32 min, s32 max)
-	{
-		return (myrand()%(max-min+1))+min;
-	}
+	s32 Rand(s32 min, s32 max);
```

Comment:
compiler should inline it itself, forcing is not always required, it's just not inlined in debug mode (sure)